.goreleaser.yml

```diff
@@ -11,4 +11,5 @@ release:
   name_template: "Flyte v{{.Version}} milestone release"
   disable: false
   extra_files:
-    - glob: ./release/*
\ No newline at end of file
+    - glob: ./release/*
+    - glob: .cr-release-packages/
```

Comment:
I don't see us copying anything into this... or is that something `cr helm...` expects/populates?

Reply:
yes, they populate the tar of helm package